packages/@sanity/migrate/src/mutations/operations/creators.ts

```diff
@@ -41,6 +52,11 @@ export const setIfMissing = <const T>(value: T): SetIfMissingOp<T> => ({
  * @returns An `unset` operation.
  * {@link https://www.sanity.io/docs/http-patches#xRtBjp8o}
  * @beta
+ *
+ * ### Example
+ * ```ts
+ * const unsetAnyValue = unset()
```

Comment:
```suggestion
 * const unsetAnyValue = unset()
 * const unsetArrayItemByIndex = unset('list[1]')
 * const unsetArrayItemByFieldValue = unset('list[_key=="foo"]')
```

Adding some more advanced examples (I _think_ they're accurate?).

Reply:
No, unset doesn't take a path, so it'd be: `at('list[_key=="foo"]', unset())`